### PR TITLE
Fixes #7443: Call rudderify from rudder upgrade

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7443

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 1]
```diff
@@ -336,6 +337,12 @@ upgrade_ncf() {
   fi
 }
 
+# Call rudderify on all local ncf techniques to make sure the promises will be properly generated
+rudderify_techniques() {
+  STEP="Call rudderify on all local ncf techniques (#7443)"
+  /usr/share/ncf/tools/ncf_rudder.py rudderify_technique /var/rudder/configuration-repository/techniques/ncf_techniques
```
Comment (VinceMacBuche) [inline rudder-webapp/SOURCES/rudder-upgrade]: Fonction called should be rudderify_techniques !

Rudderify_technique only rudderify the technique passes as parameter!!

**Comment by peckpeck** [comment]
PR updated

**Comment by VinceMacBuche** [comment]
It should also reload techniques ! Two ways:

Call reload techniques API
Put force technique reload flag file so techniques will be reloded at jetty restart

**Comment by ncharles** [comment]
Indeed, if the rudderify modify does something, it should call trigger_technique_library_reload

**Comment by peckpeck** [comment]
PR updated

**Pull request merged**